src/js/configuration/index.js

```diff
@@ -9,6 +9,7 @@ const conf = {
 	CONTENT: {
 		SERVICES_URL: readConfig('SERVICES_URL'),
 		HOMEPAGE_URL: readConfig('HOMEPAGE_URL'),
+		FORMATIONS_URL: readConfig('FORMATIONS_URL'),
```

Comment:
Minor : We should use something more english-friendly than the french `formations`. I suggest using `training` as much as possible in place of `formation`